iroh/src/node/builder.rs

```diff
@@ -316,8 +320,10 @@ where
     }
 
     /// Configure the default iroh rpc endpoint.
-    pub async fn enable_rpc(self) -> Result<Builder<D>> {
-        let (ep, actual_rpc_port) = make_rpc_endpoint(&self.secret_key, DEFAULT_RPC_PORT)?;
+    pub async fn enable_rpc(self, rpc_addr: Option<SocketAddr>) -> Result<Builder<D>> {
```

Comment:
Can we keep the default builder API simple? I would guess that 99% of all users will want to enable rpc with the default value here.

So either `.enable_rpc()` or `.enable_custom_rpc(AddrOpt)` or maybe `enable_rpc_with_opts(RpcOpts)` in case we expect more options in the future?

Reply:
fixed